content/blog/exposing-an-application-using-ingress-and-tls-termination-on-kubernetes-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,528 @@
+    Status:                True
+    Type:                  Ready
+  Not After:               2024-06-04T17:58:52Z
+  Not Before:              2024-03-06T17:58:52Z
+  Renewal Time:            2024-05-05T17:58:52Z
+  Revision:                1
+Events:                    <none>
+```
+
+### Deploy sample Nginx applications
+
+In order to configure and validate the Ingress TLS termination, three sample Nginx applications will be deployed to the cluster using the YAML manifest files from the GitHub repo [ingress-demo](https://github.com/GuopingJia/ingress-demo):
+
+```shell
+$ tree ingress-demo/
+ingress-demo/
+├── apps
+│   ├── nginx-blue.yaml
+│   ├── nginx-green.yaml
+│   └── nginx-main.yaml
+├── ingress-host-based-selfsigned.yaml
+├── ingress-path-based-selfsigned.yaml
+└── README.md
+```
+
+Each YAML manifest file in the folder *'apps'* defines the *Deployment* and the *Service* resource.  
+
+T﻿ype the following commands to deploy those Nginx applications to the namespace *nginx-apps*:
+
+```shell
+$ cd ingress-demo/
+$ kubectl apply -f apps/nginx-main.yaml -n nginx-apps
+service/nginx-main created
+deployment.apps/nginx-main created
+$ kubectl apply -f apps/nginx-green.yaml -n nginx-apps
+service/nginx-green created
+deployment.apps/nginx-green created
+$ kubectl apply -f apps/nginx-blue.yaml -n nginx-apps
+service/nginx-blue created
+deployment.apps/nginx-blue created
+```
+
+Type the command shown below to check the details of application deployment:  
+
+```shell
+$ kubectl get all -n nginx-apps
+NAME                              READY   STATUS    RESTARTS   AGE
+pod/nginx-blue-78647f4c4b-z8wq9   1/1     Running   0          10s
+pod/nginx-green-8956bbd9f-zz7hk   1/1     Running   0          22s
+pod/nginx-main-64bfd77895-tf7xd   1/1     Running   0          31s
+
+NAME                  TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)   AGE
+service/nginx-blue    ClusterIP   10.108.51.116   <none>        80/TCP    15s
+service/nginx-green   ClusterIP   10.106.115.65   <none>        80/TCP    23s
+service/nginx-main    ClusterIP   10.108.33.44    <none>        80/TCP    32s
+
+NAME                          READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/nginx-blue    1/1     1            1           15s
+deployment.apps/nginx-green   1/1     1            1           24s
+deployment.apps/nginx-main    1/1     1            1           32s
+
+NAME                                    DESIRED   CURRENT   READY   AGE
+replicaset.apps/nginx-blue-78647f4c4b   1         1         1       15s
+replicaset.apps/nginx-green-8956bbd9f   1         1         1       24s
+replicaset.apps/nginx-main-64bfd77895   1         1         1       32s
+```
+
+Three Nginx services, *nginx-main*, *nginx-blue* and *nginx-green*, are deployed as the *ClusterIP* type. They provide internal connectivity and can solely be accessed from within the cluster.
+
+T﻿ype the following commend to check that all the application service endpoints have been populated:
```

Comment:
I think you mean "command" as opposed to "commend"

Reply:
good catch !